File: threads/csharp/Seq.cs
using System;
using System.Diagnostics;
using System.Threading;

public class Seq 
{
   public static void Countdown(int n)
   {
      while (n>0)
         n -= 1;
   }

   public static void Main(string[] args)
   {
      int COUNT = 80000000;
      Stopwatch stopWatch = new Stopwatch();
      stopWatch.Start();
      Countdown(COUNT);
      stopWatch.Stop();
      TimeSpan ts = stopWatch.Elapsed;
      Console.WriteLine(ts);
   }
}

File: threads/go/seq.go
package main

import (
	"fmt"
	"time"
)

const COUNT = 80000000

func Countdown(n int) {
	for n > 0 {
		n -= 1
	}
}

func main() {
	start := time.Now()
	Countdown(COUNT)
	elapsed := time.Since(start)
	fmt.Println(elapsed)
}
